var/spack/repos/builtin/packages/parmgridgen/package.py

```diff
@@ -15,7 +15,7 @@ class Parmgridgen(Package):
     ParMGridGen is the parallel version of MGridGen.
     """
 
-    homepage = "http://www-users.cs.umn.edu/~moulitsa/software.html"
+    homepage = "https://github.com/mrklein/ParMGridGen"
```

Comment:
This one is weird. The homepage points to a Github repository that has v0.0.2 as the latest version dating 2015. The url refers to v1.0 so :man_shrugging:

Reply:
I wasn't sure what to do about this one - the original home page is 404 https://www-users.cse.umn.edu/~moulitsa/software.html. I chose the repo because it seems to be linked to those same docs: https://github.com/mrklein/ParMGridGen